packages/gatsby/src/utils/api-runner-node.js

```diff
@@ -243,13 +242,8 @@ module.exports = async (api, args = {}, pluginSource) =>
       )
     })
       .catch(err => {
```

Comment:
it seems like this catch should be for individidual plugin API call and not for `Promise.mapSeries`. It makes using `pluginName` a bit weird there (fact that it needs to be declared early) plus maybe it would allow for easier recovery in develop?

Reply:
(and also returning `null` would actually make sense there)